index.html

```diff
@@ -24,6 +24,52 @@ <h1 class="support__heading">Wspieramy Ukrainę</h1>
             </p>
         </div>
     </aside>
+    <header class="header">
+        <nav class="header__nav">
+            <div class="header__nav-left">
+                <img class="header__logo" src="./svg/hostinger.svg" alt="">
+                <img class="header__polish-flag" src="./images/flag-poland.png" alt="">
+                <a class="header__polish-text" href="">PL</a>
+            </div>
+            <div class="header__nav-right">
+                <img src="./svg/bin.svg" alt="">
+                <img src="./svg/toc_white_24dp.svg" alt="">
+            </div>
+        </nav>
+        <div class="header__bottom-place">
+            <h1 class="header__heading">
+                Najlepszy hosting dla Twojej witryny
+            </h1>
+            <p class="header__text">Do każdego rocznego planu otrzymasz bezpłatną domenę, certyfikat SSL i ochronę przed
+                atakami DDoS.
+            </p>
+            <div class="header__button-location">
+                <button class="header__button">
+                    Wybierz plan
+                </button>
+            </div>
+
+            <div class="header__text-under-button">
+                <div>
+                    <img src="./images/bird.svg" alt="">
+                    <a class="header__text-under-button-1">Wsparcie klienta 24/7/365</a>
```

Comment:
A te klasy sa w ogole potrzebne? Jesli tych tekstow by bylo 100, to bysmy mieli klase `header__text-under-button-100`. Moze lepiej byloby to zrobic w ten sposob?

```
<ul class="header-pros">
  <li class="header-pros__item">
    <img class="header-pros__image" src="./images/bird.svg" alt="">
    <a class="header-pros__text">30-dniowa gwarancja zwrotu pieniędzy</a>
  </li>
</ul>
```

Ogolnie mega lipa jest z tym `text-under-button`, co jesli sie okaze, ze teraz tekst jest nad a nie pod guzikiem? Trzeba bedzie zmieniac klase css. Warto by bylo wymyslec jakas lepsza nazwe na to